File: python_programming_basics/conditional_statements_advanced/e_01_cinema.py
movie_type = input()
rows = int(input())
cols = int(input())
result = 0

area = rows * cols

if movie_type == "Premiere":
    result = area * 12

elif movie_type == "Normal":
    result = area * 7.5

elif movie_type == "Discount":
    result = area * 5

print(f"{result:.2f} leva")

File: JavaScript Basics/03. Conditional Statements Advanced/e-01-Cinema.js
function calculateIncome(projectionType, rows, columns) {
    let income = 0;
    let totalSeats = rows * columns;

    if (projectionType === "Premiere") {
        income = totalSeats * 12.00;
    } else if (projectionType === "Normal") {
        income = totalSeats * 7.50;
    } else if (projectionType === "Discount") {
        income = totalSeats * 5.00;
    } else {
        console.log("Invalid projection type");
        return;
    }

    console.log(`${income.toFixed(2)} leva`);
}

calculateIncome("Premiere", 10, 12);
calculateIncome("Normal", 21, 13);
calculateIncome("Discount", 12, 30);
